### PR TITLE
Version bump to 5.6.5

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

**Comment by andrewvc** [comment]
LGTM!

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
+* <<logstash-5-6-5,Logstash 5.6.5>>
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-5]]
+=== Logstash 5.6.5 Release Notes
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]: any thoughts of having a note here about why this release is happening?

Comment (robbavey) [reply docs/static/releasenotes.asciidoc]: This is a placeholder for 5.6.5 release if and when it happens

Comment (robbavey) [reply docs/static/releasenotes.asciidoc]: @jsvd @andrewvc On reflection, I'm going to remove the reference to 5.6.5 here, and wait until if and when a release comes

**Comment by mgreau** [comment]
Hello
It seems that this PR updates the logstash version of the documentation to `5.6.5` at:

-  https://www.elastic.co/guide/en/logstash/5.6/docker.html

It results in a wrong documentation because the `docker.elastic.co/logstash/logstash:5.6.5` doesn't exist:

> The Docker image for Logstash 5.6.5 can be retrieved with the following command:
>   `docker pull docker.elastic.co/logstash/logstash:5.6.5
`

**Comment by robbavey** [comment]
@mgreau This should be fixed by https://github.com/elastic/logstash/pull/8638

**Pull request closed without merging**